scripts_future_API/test_levelsets_volumefractions.jl

```diff
@@ -0,0 +1,109 @@
+using CUDA
+using JLD2
+using FastIce.LevelSets
+using Chmy.Grids
+using Chmy.Fields
+using Chmy.Architectures
+
+
+# Data
+# vavilov_path = "../data/vavilov.jld2"
+# vavilov_path = "../data/Vavilov/Vavilov_80m.jld2"
+vavilov_path = "../data/Vavilov/Vavilov_500m.jld2"
+# synthetic_data = "../data/Synthetic/dome_glacier.jld2"
+# synthetic_data = "../data/Synthetic/low_res_dome_glacier.jld2"
+
+# Select backend
+backend = CPU()
+# backend = CUDABackend()
+arch = Arch(backend)
+
+"""
+    load_dem_on_GPU(backend, arch::Architecture, path::String)
```

Comment:
why "on_GPU"? The function loads DEM on a given backend. SO I would call then function simply load_dem, or load_data, load_topo, or else.